HOW_WE_USE_GITHUB.md

```diff
@@ -136,14 +138,38 @@ Once issues are deemed ready to be worked on, they will be moved to the [`conda`
 
 #### How does labeling work?
 
 Labeling is a very important means for sorting engineers to keep track of the current state of an issue with regards to the asynchronous nature of communicating with users. Utilizing the proper labels helps to identify the severity of the issue as well as to quickly understand the current state of a discussion.
 
-Generally speaking, labels with the same category are considered mutually exclusive but in some cases labels sharing the same category can occur concurrently as they indicate qualifiers as opposed to types. For example, we may have the following types, https://github.com/conda/infra/labels/type%3A%3Abug, https://github.com/conda/infra/labels/type%3A%3Afeature, and https://github.com/conda/infra/labels/type%3A%3Adocumentation, where for any one issue there would be _at most_ **one** of these to be defined (_i.e._ an issue shouldn’t be a bug _and_ a feature request at the same time). Alternatively, with issues involving specific operating systems (_i.e._, https://github.com/conda/infra/labels/os%3A%3Alinux, https://github.com/conda/infra/labels/os%3A%3Amacos, and https://github.com/conda/infra/labels/os%3A%3Awindows), an issue could be labeled with one or more depending on the system(s) the issue is occurring on.
+Each label has an associated description that clarifies how the label should be used. To see a label's description, find the label either in `conda/infrastructure`'s [`.github/global.yml` file][labels-global] or, if it is not there, in the relevant repository's `.github/labels.yml` file. Label colors are used to distinguish labels by category.
+
+Generally speaking, labels with the same category are considered mutually exclusive but in some cases labels sharing the same category can occur concurrently as they indicate qualifiers as opposed to types. For example, we may have the following types, https://github.com/conda/infrastructure/labels/type%3A%3Abug, https://github.com/conda/infrastructure/labels/type%3A%3Afeature, and https://github.com/conda/infrastructure/labels/type%3A%3Adocumentation, where for any one issue there would be _at most_ **one** of these to be defined (_i.e._ an issue shouldn’t be a bug _and_ a feature request at the same time). Alternatively, with issues involving specific operating systems (_i.e._, https://github.com/conda/infrastructure/labels/os%3A%3Alinux, https://github.com/conda/infrastructure/labels/os%3A%3Amacos, and https://github.com/conda/infrastructure/labels/os%3A%3Awindows), an issue could be labeled with one or more depending on the system(s) the issue is occurring on.
+
+Please note that there are also automation policies in place. For example, if an issue is labeled as https://github.com/conda/infrastructure/labels/pending%3A%3Afeedback and https://github.com/conda/infrastructure/labels/unreproducible, that issue will be auto-closed after a month of inactivity.
```

Comment:
This isn't correct. Any issue labeled as https://github.com/conda/infrastructure/labels/type%3A%3Asupport is closed after a month of inactivity.

Reply:
Maybe it would be useful to link to the GitHub action that actually contains the logic for all this?

Reply:
OK, so I'm getting from this that:
- regardless of any other label, all issues labelled [type::support](https://github.com/conda/infrastructure/labels/type%3A%3Asupport) will be closed after a month of inactivity, and
- we have additional labels that might be optionally used to indicate why the issue needs support, but this document shouldn't mention them.

Please confirm if this understanding is correct.

Reply:
@kalawac, after reading through the GitHub workflow we have defined to determine this, it looks like `type::support` are first labeled "stale" after 21 days of inactivity. Once they have been "stale" for 7 days, they are then closed.

Reference: https://github.com/conda/infrastructure/blob/main/.github/workflows/stale.yml#L23-L36

Regarding the second question, issue sorters are free to use whatever other labels they deem necessary. I don't think it's worth mentioning here though.